processing/src/main/java/io/druid/query/groupby/GroupByQueryConfig.java

```diff
@@ -66,6 +67,12 @@
   // Max on-disk temporary storage, per-query; when exceeded, the query fails
   private long maxOnDiskStorage = 0L;
 
+  @JsonProperty
+  private boolean forcePushDownLimit = false;
+
+  @JsonProperty
+  private Class<? extends GroupByQueryMetricsFactory> queryMetricsFactory;
```

Comment:
@leventov Patch to remove those is here, https://github.com/druid-io/druid/pull/4383, thanks for catching that